sdk/schemaregistry/azure-data-schemaregistry-apacheavro/CHANGELOG.md

```diff
@@ -76,15 +75,6 @@
 - Upgraded `azure-data-schemaregistry` from `1.3.2` to version `1.3.3`.
 - Upgraded `azure-core` from `1.35.0` to version `1.36.0`.
 
-## 1.2.0-beta.2 (2023-02-13)
```

Comment:
is this removed by purpose?

Reply:
That was not intentional. Thanks for getting that.